### DISCUSSION TITLE
Please can the American Network News Channels be separated from their entertainment premium counterparts.

### DISCUSSION THREAD

**Discussion by paul19920801** [iptv-org/community#77] [Ideas]
Hi all

Is there any way that the USA FTA channels such as ABC/CBS/FOX etc which carry the actual syndicated programming be separated from the News variants of the same name which simply carries news. I know those premium network channels are there, but are buried with the other news, and quite frankly for someone who doesn't live in the USA, the other stuff is rubbish. 

I am trying to separate them myself but half the time the networks just carry news, which is the same as both channels. 

Please can someone do this.

Thanks

**Comment by BellezaEmporium** [comment]
We'll need to study this case, since we do not know why we would split those.

**Comment by what-a** [comment]
I'm thinking it's probably best to address this by a change in their name rather than splitting them out. For example, add "newscast" to those that are news only. Thinking about this some more... The terms used by the TV news industry are "over the top" vs. "over the air". Should we add ATT vs. OTA?
OTT = internet broadcast (i.e. newcast only)
OTA = full regional over the air broadcast
